spec/spec_helper.rb

```diff
@@ -33,3 +33,13 @@
     end
   end
 end
+
+def select_io_pair_by_name(category, name)
+  JsonApi::Parameters::Testing::PAIRS[category].find do |example|
+    example.key?(name)
+  end[name].deep_dup
+end
+
+def select_input_by_name(category, name)
+  select_io_pair_by_name(category, name)[0]
+end
```

Comment:
No new line on end of file. This should be reported by rubocop?

Reply:
Good catch, turns out rubocop had spec_helper.rb excluded from all cops. Fixing that!